### PR TITLE
Update Kubernetes and kubernetes-cni versions

### PR DESCRIPTION
**Pull request by isabelmsft**
This PR updates kubernetes version to 1.18.6 and  kubernetes-cni version to 0.8.6

signed-off by: Isabel Li isabel.li@microsoft.com

- **Why I did it**
Previous kubernetes-cni version (0.7.5) introduced Kubernetes Man In The Middle Vulnerability. “A vulnerability was found in all versions of containernetworking/plugins before version 0.8.6, that allows malicious containers in Kubernetes clusters to perform man-in-the-middle (MitM) attacks. A malicious container can exploit this flaw by sending rogue IPv6 router advertisements to the host or other containers, to redirect traffic to the malicious container.”

- **How I did it**
Defined kubernetes-cni version to be 0.8.6 and updated kubernetes version to be 1.18.6

- **How to verify it**
Check versions by running `dpkg -l | grep kube`

### CODE REVIEW

File: Makefile.work [hunk 1]
```diff
@@ -185,6 +185,7 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            SHUTDOWN_BGP_ON_START=$(SHUTDOWN_BGP_ON_START) \
                            INSTALL_KUBERNETES=$(INSTALL_KUBERNETES) \
                            KUBERNETES_VERSION=$(KUBERNETES_VERSION) \
+			   KUBERNETES_CNI_VERSION=$(KUBERNETES_CNI_VERSION) \
```
Comment (lguohan) [inline Makefile.work:188]: tabs to spaces.

Comment (isabelmsft) [reply Makefile.work:188]: Spacing seems correct in other views, but it looks off in the code snippet from PR review

My local file:
image
Clicking on 'View file' on GitHub: 
image

Comment (renukamanavalan) [reply Makefile.work:188]: Open in vi and search for tabs. (Type / followed by cntrl-v & cntrl-i to search for tabs)

Comment (isabelmsft) [reply Makefile.work]: Thank you, it is fixed now

**Comment by qiluo-msft** [comment]
Retest mellanox please

**Comment by isabelmsft** [comment]
retest mellanox please

**Comment by isabelmsft** [comment]
retest baseimage please

**Pull request merged**